honeybee_energy/hvac/_base.py

```diff
@@ -0,0 +1,129 @@
+# coding=utf-8
+"""Base class to be used for all HVAC systems."""
+from __future__ import division
+
+from ..schedule.ruleset import ScheduleRuleset
+from ..schedule.fixedinterval import ScheduleFixedInterval
+
+from honeybee._lockable import lockable
+from honeybee.typing import valid_ep_string
+
+from os.path import dirname, basename, isfile, join
+from os import listdir
+from importlib import import_module
```

Comment:
So what do you recommend, then?  I guess that I can remove the ability to extend the `hvac` subpackage by dropping hvac modules into the folder since I realized that the way that I was planning to extend the subpackage with the future `honeybee-energy-hvac` repo isn't necessarily reliant on this.  In this case, I will hard code the fact that `IdealAirSystem` is the only HVAC system that comes with `honeybee_energy`. Would you be in support of this?

On another note, [the other place where we use this strategy is over in `ladybug.datatype`](https://github.com/ladybug-tools/ladybug/blob/master/ladybug/datatype/base.py#L7). In that case, though, we are really reliant on it in order to make all of the datatype modules nice and small and make the whole subpackage easily extensible with new data types. Please open an issue over there if you have a recommended alternative.

Reply:
Not sure how this reply is related to the original comment about importing for os.

I believe this is in relation to creating the dictionary which I replied to below.

Reply:
Hm.  Now I'm less sure what you are commenting on. Were you referring to the `from os.path import dirname, basename, isfile, join` or the `from importlib import import_module`?

I mean, all of this importing is going to get removed because of the dictionary comment but it would still be good to know what you mean here for future reference and for the sake of ladybug.

Reply:
I meant mainly for the os module. Not to use `from * import x, y, z`.